.github/workflows/release-pypi.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Release to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_sdist:
+    name: Build source package
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install/Upgrade Python dependencies
+      run: python -m pip install --upgrade pip wheel
+
+    - name: Build and install Python source package
+      run: |
+        python setup.py sdist
+        python -m pip install dist/*.tar.gz
+
+    - name: Run tests
+      run: |
+        pip install pytest pytest-xdist
+        pytest -n auto test/
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: dist/*.tar.gz
+
+
+  build_bdist:
+    name: Build binary wheels
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Windows build fails
```

Comment:
See: https://github.com/dralley/hpy/runs/3287054071?check_suite_focus=true#step:4:218

Reply:
I'm confused. Windows tests pass but windows build fails? That's curious, but not a merge blocker of course.

Reply:
It seems to have been resolved in the meantime?  But it looks like there is maybe another issue.

https://github.com/dralley/hpy/runs/3551010010?check_suite_focus=true#step:4:265

Reply:
I'm fine to skip windows build and merge the PR ASAP. Then we can fix windows later